charts/otomi-api/templates/serviceaccount.yaml

```diff
@@ -0,0 +1,21 @@
+
+apiVersion: v1
+kind: ServiceAccount
+{{- with .Values.imagePullSecrets }}
+imagePullSecrets:
+  {{- toYaml . | nindent 2 }}
+{{- end }}
+
+automountServiceAccountToken: true
+metadata:
+  name: {{ template "otomi-api.serviceAccountName" . }}
+  namespace: {{ .Release.Namespace | quote }}
+  {{- with .Values.serviceAccount.annotations }}
+  annotations:
+    {{- toYaml . | nindent 4 }}
+  {{- end }}
+  labels:
+    {{- include "otomi-api.labels" . | nindent 4 }}
+    {{- with .Values.serviceAccount.labels }}
```

Comment:
question: don't these need to also exist in the chart's values? I think helm 3 uses the values file as a sort of schema to determine what is expected, and throws an error if other stuff is provided. I am referring to these missing values:
- `.Values.serviceAccount.annotations`
- `.Values.serviceAccount.labels`

Reply:
helm3 does not throw error, however I can add those to values.